FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +20,26 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
+
+			switch result {
+			case .success(let (data, response)):
+
+				guard response.statusCode == self.OK_200 else {
+					return completion(.failure(Error.invalidData))
+				}
+
+				guard let root = try? JSONDecoder().decode(Root.self, from: data) else {
+					return completion(.failure(Error.invalidData))
+				}
```

Comment:
Since both `guard` statements complete with the same `Error`, consider joining them:
```suggestion
				guard response.statusCode == self.OK_200, let root = try? JSONDecoder().decode(Root.self, from: data) else {
					return completion(.failure(Error.invalidData))
				}
```

Reply:
In the future I would like to catch each error separately. But for now there is no use case so joining seems fine 👌🏽